packages/server/lib/makeDataContext.ts

```diff
@@ -27,6 +27,9 @@ export function makeDataContext (options: MakeDataContextOptions) {
       getBrowsers () {
```

Comment:
Seems like rather than declaring wrappers around our imports, we could just do

```
appApi: {
  getBrowsers,
  ensureAndGetByNameOrPath,
}
```

(same elsewhere in this file)